cli/main.rs

```diff
@@ -218,6 +218,7 @@ pub fn create_main_worker(
     // above
     ops::errors::init(js_runtime);
     ops::runtime_compiler::init(js_runtime);
+    ops::test_runner::init(js_runtime);
```

Comment:
These ops should only be added if we're in `deno test` - I guess adding a param to this callback is sufficient